app/Livewire/Components/Modules/Maintenance/FixJobs.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Livewire\Components\Modules\Maintenance;
+
+use App\Enum\JobStatus;
+use App\Models\Configs;
+use App\Models\JobHistory;
+use App\Policies\SettingsPolicy;
+use Illuminate\Support\Facades\Gate;
+use Illuminate\View\View;
+use Livewire\Component;
+
+/**
+ * Sometimes the job history is a bit messed up,
+ * this happens when there are crashes or error in the logic.
+ *
+ * In theory this should not be needed but if this is not resolved
+ * the pulse feedback would always stay alive.
+ */
+class FixJobs extends Component
+{
+	private array $waitingJobsTypes = [JobStatus::READY, JobStatus::STARTED];
+
+	/**
+	 * Rendering of the front-end.
+	 *
+	 * @return View
+	 */
+	public function render(): View
+	{
+		Gate::authorize(SettingsPolicy::CAN_UPDATE, Configs::class);
+
+		return view('livewire.modules.maintenance.fix-jobs');
+	}
+
+	/**
+	 * Return the size used by Lychee.
+	 * We now separate this from the initial get() call as this is quite time consuming.
```

Comment:
This description looks wrong?

Reply:
indeed.

Reply:
Fixed: [8695d4c](https://github.com/LycheeOrg/Lychee/pull/2352/commits/8695d4cf7e9fb1750ecd308b3fcb2a8a7b0bf62e)